src/App.css

```diff
@@ -1,22 +1,22 @@
-.App {
-  text-align: center;
+* {
```

Comment:
are we going to stick with vanilla css for styling?

Reply:
Hey Eric,
styling will be addressed in the next phase of the sprint. Not an issue now